content/index.rst

```diff
@@ -18,10 +19,120 @@ summary: |
   .. callmacro:: content.macros.j2#link_button
     :url: '/contact/'
 
-    Let's talk about your idea
+    Get a free consultation
 
   .. _web application design and development: /work/
 
 
 Software for Humans
 ===================
+
+
+.. callmacro:: content.macros.j2#get_quotes
+  :slug: 'work/coachhub'
+  :index: 1
+
+.. callmacro:: projects/splash.macros.j2#splash_list
+
+
+.. ---------------------------------
+.. callmacro:: content.macros.j2#rst
+  :tag: 'start'
+
+.. Don't mess with typography...
+.. wrap:: div
+
+  .. callmacro:: content.macros.j2#heading
+    :content: 'Small Team, Large Expertise'
+    :subhead: 'Perfect for funded startups, and enterprise augmentation'
+
+  Founded by a `Django core developer`_
+  and the `creator of Susy`_ responsive-layout framework,
+  we provide industry-leading expertise
+  on a small team –
+  completely focused on the needs of your company and users,
+  while integrating design and development
+  in an agile process.
+  *Big agency skill,
+  with the attention and agility
+  of an internal team.*
+
+  .. _Django core developer: /birds/#bird-carl
+  .. _creator of Susy: /birds/#bird-miriam
+
+  .. callmacro:: content.macros.j2#link_button
+    :url: '/contact/'
+
+    Start a conversation
+
+.. callmacro:: content.macros.j2#rst
+  :tag: 'end'
+.. ---------------------------------
+
+
+.. callmacro:: content.macros.j2#get_quotes
+  :slug: 'work/moztrap'
+  :index: 1
+
+.. callmacro:: content.macros.j2#get_quotes
+  :slug: 'work/quarqnet'
+  :index: 1
+
+.. callmacro:: projects/splash.macros.j2#splash_list
+  :slugs: [
+    'open-source/susy',
+    'open-source/django',
+    'open-source/true',
+    'open-source/plone'
+    ]
+
+.. callmacro:: content.macros.j2#get_quotes
+  :slug: 'open-source/susy'
+  :index: 1
+
+.. ---------------------------------
+.. callmacro:: content.macros.j2#rst
+  :tag: 'start'
+
+Book: Jump Start Sass
```

Comment:
I feel like this needs a more defined section separator, when we have created those...